cli/ask_confirm.go

```diff
@@ -23,38 +23,58 @@ func AskForConfirmation(s string) bool {
 
 		response = strings.ToLower(strings.TrimSpace(response))
 
-		if response == "y" || response == "yes" {
+		if response == "Y" || response == "y" || response == "yes" {
 			return true
 		} else if response == "n" || response == "no" {
 			return false
+		} else if response == "" {
+			return true
 		}
 	}
 }
 
 // MultiChoice for multiple choices question. It returns the selected option
-func MultiChoice(s string, opts ...string) string {
+func MultiChoice(s string, opts ...string) int {
 	reader := bufio.NewReader(os.Stdin)
 
 	fmt.Println(s)
+	if len(opts) == 0 {
+		log.Fatal(fmt.Errorf("no choice available"))
+	}
 	for i, o := range opts {
```

Comment:
just a comment for another PR: we should handle a kind of "pages" and display 20 options per 20 options...

Reply:
issue created